### PR TITLE
#627 - Limit upcoming events

### PR DESCRIPTION
**Pull request by yasirazgar**
Issue - https://github.com/ReadyResponder/ReadyResponder/issues/627

### CODE REVIEW

File: app/services/msg/upcoming.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 class Msg::Upcoming < Msg::Base
   # method called with: @params (From: String, Body: String), @person (Person Object)
-  def respond
-    events = Event.upcoming.limit(10)
```
Comment (yasirazgar) [inline app/services/msg/upcoming.rb:4]: removed this unused variable,please let me know if I am missing something.

**Comment by kgf** [comment]
I like how you've moved it into the person model. 

I don't know why the tests are failing.

**Comment by kgf** [comment]
I've updated development to get a more recent Chrome driver. Let's see if that resolves the Travis errors...

**Comment by kgf** [comment]
@yasirazgar Thanks for your help on this, it is appreciated !

Might you be tempted by #632, which is in the same area ?

**Comment by yasirazgar** [comment]
Yup, will look into #632, thanks! @kgf

**Pull request merged**